envoy/rockcraft.yaml

```diff
@@ -0,0 +1,52 @@
+# Based on: https://github.com/envoyproxy/envoy/blob/release/v1.27/ci/Dockerfile-envoy
+name: envoy
+summary: Envoy is an open-source edge and service proxy designed for cloud-native applications.
+description: |
+  Envoy is essential for modern applications, particularly those built using microservices, 
+  by providing reliable, efficient, and secure service communication along with valuable observability features.
+version: "1.27.0"
+license: Apache-2.0
+base: ubuntu@22.04
+run-user: _daemon_
+
+services:
+  envoy:
+    override: replace
+    summary: "Entry point for envoy oci-image"
+    startup: enabled
+    command: "/envoy -c /envoyproxy_io_proxy.yaml"
+    user: envoy
+
+platforms:
+  amd64:
+
+parts:
+  envoy:
+    plugin: nil
+    source: https://github.com/envoyproxy/envoy.git
+    source-tag: release/v1.27
+    build-packages:
+      - curl
+    build-environment:
+      - ENVOY_VERSION: 1.27.0
+    override-build: |
+      cp $CRAFT_PART_SRC/configs/envoyproxy_io_proxy.yaml $CRAFT_PART_INSTALL
+      curl -L https://github.com/envoyproxy/envoy/releases/download/v${ENVOY_VERSION}/envoy-${ENVOY_VERSION}-linux-x86_64 \
```

Comment:
Where does this `curl` command correspond? I don't see any command like this in the upstream [dockerfile](https://github.com/envoyproxy/envoy/blob/main/ci/Dockerfile-envoy#L48)?

Reply:
Hey @orfeas-k! Yes, but Kimonas mentioned that we don't need build any artefacts here, just copy binary. Here with curl I just copy binary with specific version from release page and run it, isn't it solve this case?